src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        setColor(color);
+        this.radius = radius;
+    }
+
+    public Circle(String color, int radius, String name) {
+        setName(name);
+        setColor(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure name: circle, color: "
+                + getColor() + " radius: " + radius);

```

Comment:
The `draw()` method should not be responsible for outputting information directly to the console. Consider using a method that returns a String representation of the object, and let another class handle the output.